.github/workflows/shiplog-sign.yml

```
@@ -0,0 +1,97 @@
+          [ -n "${CI_SSH_PUBLIC_KEY:-}" ] || { echo "Missing CI_SSH_PUBLIC_KEY env" >&2; exit 1; }
+          umask 077
+          mkdir -p ~/.ssh
+          printf '%s\n' "$CI_SSH_PRIVATE_KEY" > ~/.ssh/id_shiplog_ci_ed25519
+          printf '%s\n' "$CI_SSH_PUBLIC_KEY" > ~/.ssh/id_shiplog_ci_ed25519.pub
+          git config --global gpg.format ssh
+          git config --global user.signingkey ~/.ssh/id_shiplog_ci_ed25519
+          git config --global user.name "Sea Eye"
+          git config --global user.email "ci@flyingrobots.dev"
+
+      - name: Prepare Shiplog environment
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:5:42: Double quote to prevent globbing and word splitting

(shellcheck)